libafl_qemu/src/lib.rs

```diff
@@ -7,6 +7,7 @@
     allow(clippy::useless_conversion)
 )]
 #![allow(clippy::needless_pass_by_value)]
+#![allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
That's a false positive? Always?

Reply:
In libafl_qemu, it is at multiple points. I'll make this statement-level allows.